exetera/core/fields.py

```diff
@@ -143,6 +143,27 @@ def _ensure_valid(self):
         if not self._valid_reference:
             raise ValueError("This field no longer refers to a valid underlying field object")
 
+    def __getitem__(self, item:Union[list, tuple, np.ndarray]):
+        if isinstance(item, slice):
+            # TODO
+            pass
+        elif isinstance(item, int):
+            # TODO
```

Comment:
You can get an array of a single value with `self.data[[item]]` and then put that into a mem field.